src/css/Login.css

```diff
@@ -0,0 +1,36 @@
+.h_size {
+  font-size: 40px !important;
+  font-weight: 30 !important;
+}
+
+/* Margin */
+.margin_s {
+  padding-left: 9em;
+  padding-top: 18em;
+}
+
+.margin_s h3 {
+  font-weight: 20;
+}
+
+.margin_s h5 {
+  font-weight: 20;
+  padding-top: 10px;
+  font-size: 18px;
+  padding-bottom: 20px;
+}
+
+.margin_second {
```

Comment:
It would be best to have a more descriptive CSS class name, as if you aren't using [CSS modules](https://css-tricks.com/css-modules-part-1-need/) or styled components, CSS is scoped globally by default in React.